README.md

```diff
@@ -4,22 +4,25 @@ A Fluent filter plugin to anonymize records which have PAN (Primary Account Numb
 
 Inspired by [fluent-plugin-anonymizer](https://github.com/y-ken/fluent-plugin-anonymizer).
 
+**N.B.:** This fork adds ability to allow Regex capture group usage, so that you can mask partially. The configuration shows how to set first 6 and last 4 numbers available while masking the values in between. See the example below.
+
 # Requirements
 
 - fluentd: v0.14.x or later
 - Ruby: 2.4 or later
 
 # Installation
 
-```
-gem install fluent-plugin-pan-anonymizer
+```shell
+fluent-gem install specific_install
+fluent-gem specific_install https://github.com/zbalkan/fluent-plugin-pan-anonymizer.git
 ```
```

Comment:
ditto

Reply:
Reverted. Kept the changes in a another branch for future.